shell/browser/native_window_views.cc

```diff
@@ -442,6 +442,10 @@ void NativeWindowViews::Hide() {
   if (!features::IsUsingOzonePlatform() && global_menu_bar_)
     global_menu_bar_->OnWindowUnmapped();
 #endif
+
+#if defined(OS_WIN)
+  taskbar_host_.SetThumbarButtonsAdded(false);
```

Comment:
Can you add comment for this line? Without context it is hard to understand why this call is needed.